codes/AppEndpointType.ttl

```diff
@@ -38,4 +38,10 @@ idsc:USAGE_POLICY_ENDPOINT
     a ids:AppEndpointType;
     rdfs:label "Usage policy endpoint"@en ;
     rdfs:comment "Endpoint is used for usage policy / usage control related scenarios."@en ;
+.
+
+idsc:SELF_DESCRIPTION_ENDPOINT
+    a ids:AppEndpointType;
+    rdfs:label "Self description endpoint"@en ;
```

Comment:
Please use either "self-description" or "self description" in both label and comment. We should be consistent.